streams/src/main/java/org/apache/kafka/streams/Consumed.java

```diff
@@ -24,10 +24,29 @@
 
 /**
  * The {@code Consumed} class is used to define the optional parameters when using {@link StreamsBuilder} to
- * build instancs of {@link KStream}, {@link KTable}, and {@link GlobalKTable}.
+ * build instances of {@link KStream}, {@link KTable}, and {@link GlobalKTable}.
+ * <p>
+ * For example, you can read an topic as {@link KStream} with custom timestamp extractor and specify the corresponding
+ * key and value serdes like:
+ * <pre>{@code
+ * StreamsBuilder builder = new StreamsBuilder();
+ * KStream<String, Long> stream = builder.stream(
+ *   "topicName",
+ *   Consumed.with(Serdes.String(), Serdes.Long())
+ *           .withTimestampExtractor(new LogAndSkipOnInvalidTimestamp()));
+ * }</pre>
+ * Similar, you can read a topic as {@link KTable} with custom "auto.offset.reset" strategy and force a state store
```

Comment:
Typo:
> Similar[ly], you can ... with [a] custom ...

Reply:
Consistency: Here we say

> with [a] custom "auto.offset.reset" strategy

and below we say 

> Enum used to define auto offset reset policy when creating {@link KStream} or {@link KTable}.

The AK docs say neither "strategy" or "policy", it's just a config like any other.  Hence I'd suggest to change the above consistently to sth like:

> with [a] custom {@code auto.offset.reset} configuration

and

> Sets the {@code auto.offset.reset} configuration when creating {@link KStream} or {@link KTable}.